.github/workflows/ci.yml

```diff
@@ -67,7 +67,7 @@ jobs:
           flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
 
       - name: Test with pytest
-        run: pytest tests
+        run: pytest tests --cov=aif360 --cov-report=term-missing --cov-fail-under=80
```

Comment:
Let's start small and just report the coverage instead of setting a failure criterion -- mainly because we're failing right now 🙂

Eventually we can add it back and/or upload to codecov once we add more tests

Reply:
Thanks for the comments! I just pushed a couple commits adding `pytest-cov` to `setup.py` and removing the failure criterion.